src/lib/auth.js

```diff
@@ -31,6 +33,16 @@ class Auth implements IAuth {
     this.secret = config.secret;
     this.plugins = this._loadPlugin(config);
     this._applyDefaultPlugins();
+
+    this.plugins.forEach((plugin) => {
+        if (plugin.login_url) {
+            if (this.login_url && this.login_url != plugin.login_url) {
+                throw new Error('conflicting auth plugins: multiple plugins tried to set different login urls');
+            }
+
+            this.login_url = plugin.login_url;
```

Comment:
This should be part of the plugins dev documentation

Reply:
:+1: